views/download.handlebars

```diff
@@ -35,5 +35,5 @@
     </div>
   </div>
 
-  <a class="send-new" data-l10n-id="sendYourFilesLink" target="_blank"></a>
+  <a class="send-new" data-l10n-id="sendYourFilesLink" href="/"></a>
```

Comment:
Didn't we have the `target="_blank"` there to prevent disrupting a download? #398